src/js/components/inspectors/Scale.tsx

```diff
@@ -20,22 +21,41 @@ interface StateProps {
 
 
 function mapStateToProps(state: State, ownProps: OwnProps): StateProps {
+  console.log(getInVis(state, 'scales.' + ownProps.primId))
   return {
     scale: getInVis(state, 'scales.' + ownProps.primId)
   };
 }
 
 class BaseScaleInspector extends React.Component<OwnProps & StateProps> {
+
   public render() {
-    const scale = this.props.scale;
+    const props = this.props;
+    const scale = props.scale;
+    const typeOpts = ['linear', 'log', 'time', 'ordinal', 'point'];
     return (
       <div>
         <div className='property-group'>
           <h3 className='label'>Placeholder</h3>
           <ul>
             <li>name: {scale.get('name')}</li>
             <li>type: {scale.get('type')}</li>
-            {/* <li>range: {scale.get('range')}</li> TODO */}
+
+            <Property name='type' label='Type' type='select' opts={typeOpts} {...props} />
```

Comment:
I'm not sure how the onChange works here, doesn't seem to be updating the property